logger/logger.go

```diff
@@ -84,12 +79,25 @@ type Logger interface {
 	Traceln(v ...interface{})
```

Comment:
Do we still need to maintain this logging interface? Since we use logrus anyway, I would recommend to remove it or use an alias to the logrus field logger:
```go
type Logger logrus.FieldLogger
```

Reply:
The interface was created to have a possibility to replace the logger with something else in unit tests in order to test if host-metering logs as expected.  I don't think it was used much.  When I was looking into logrus doc, I noticed there is a hooks feature( https://pkg.go.dev/github.com/sirupsen/logrus#Hook ) which seems as something that can provided similar capability - if needed.